03week/towersOfHanoi.js

```diff
@@ -19,24 +19,56 @@ function printStacks() {
   console.log("c: " + stacks.c);
 }
 
-function movePiece() {
+function movePiece(startStack,endStack) {
   // Your code here
-
+  const startArray = getStackArrays(startStack)
+  const endArray = getStackArrays(endStack)
+  const targetPiece = startArray.pop()
+  endArray.push(targetPiece);
+  console.log(targetPiece);
+  console.log(endArray)
 }
 
-function isLegal() {
+function getStackArrays(targetStack){
```

Comment:
This function is a little redundant. For example, on line 24 you could simply run:

`startArray = stacks[startStack]`